docs/src/pages/docs/workflows/typescript.mdx

```diff
@@ -115,7 +115,7 @@ declare global {
 If you're encountering problems, double check that:
 
 1. Your interface uses the correct name.
-2. You're using TypeScript version 4 or later.
+2. You're using TypeScript version 5 or later.
```

Comment:
Sounds fair to me. Typed messages theoretically only requires TypeScript 4, but if we mention a TypeScript version, it's probably a good idea to mention one that supports all features from `next-intl` 👍.